GameServerLib/Logging/CustomColoredConsoleAppender.cs

```diff
@@ -0,0 +1,672 @@
+﻿#region Apache License
```

Comment:
@MythicManiac is this k?

Reply:
I have no idea, so I would stay away from multilicensing if possible